### ISSUE TITLE
Progress Report 1

### ISSUE THREAD

**Issue by jordan-heemskerk** [RobertLeahy/SENG499#4]

**Comment by jordan-heemskerk** [comment]
The first progress report should clearly define the problems associated with the project, the
team members involved (“company”- with contact information), the intended scope of the
project, the proposed solution(s), the assigned tasks, the anticipated milestones and progress
made (if any).
It should also include a project summary to be used for the Poster Presentation in layman’s
language. 
3
REQUIRED INFORMATION IN THE FIRST PROGRESS REPORT
Project Number: (to be assigned)
Project Title: (selected by the team and agreed by the Supervisor)
Personnel (to include email address and mobile phone of a contact person):
Faculty Supervisor’s name and Department
Project Summary (approximately 200 words) – please use simple language

**Comment by jordan-heemskerk** [comment]
Due June 2

**Comment by jordan-heemskerk** [comment]
@4444jw4444 you OK to take this on?

**Comment by JorinDW** [comment]
@jordan-heemskerk yeah, I will keep you guys posted on progress and maybe ask some questions about what we would like to include.

**Comment by jordan-heemskerk** [comment]
Sounds good!
